.changeset/tsconfig-dangerously.md

```diff
@@ -0,0 +1,12 @@
+---
+'sku': minor
+---
+
+`srcPaths` no longer affects `tsconfig.json#include`. Instead, you can use [`dangerouslySetTSConfig`][dangerous] option to have more control over which files are included in the TypeScript compilation process.
+
+Previously, sku managed the `include` field in `tsconfig.json`, but this was problematic for projects that wanted more fine grained control over what is included and/or excluded from compilation.
```

Comment:
I _think_ this should be `was` so it matches the tense of the rest of the sentence, e.g. `wanted`.
```suggestion
Previously, sku managed the `include` field in `tsconfig.json`, but this was problematic for projects that wanted more fine grained control over what was included and/or excluded from compilation.
```